Shell/TheoryFinder.cpp

```diff
@@ -418,6 +424,7 @@ bool TheoryFinder::matchCode(const void* obj,
 #endif
     unsigned l = code[cp+1];
     // bit field of choices for this literal
+    ASS(clength_assigned);
     unsigned choice = (1u << clength) - 1;
```

Comment:
Same as with `kept_assigned` above - really has to be initialised otherwise something bad will happen.